src/components/tasks/card/progressBar/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC, useState } from 'react';
+import { useRouter } from 'next/router';
+
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import {
+    useUpdateSelfTaskMutation,
+    useUpdateTaskMutation,
+} from '@/app/services/tasksApi';
+import useUserData from '@/hooks/useUserData';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import HandleProgressText from './ProgressText';
+import HandleProgressbar from './ProgressBar';
+import { ERROR_MESSAGE, PROGRESS_SUCCESSFUL } from '@/constants/constants';
+
+import classNames from '@/components/tasks/card/card.module.scss';
+import { ProgressContainerProps } from '@/interfaces/task.type';
```

Comment:
why do all imports follow vertical spacing of 1 line, any specific reason? it would be better to keep it consistent without spaces like other files for better readability!

Reply:
with spacing you can provide better readability and spacing here specifies the one kind of imports like from line 1 to 2 is the libraries we are going to use, from line 4 t 10 specifies the rtk query and custom hooks, and so on and there is no harm in giving spacing.

Reply:
replied in the below comment